docs/logs/discovery/tailing-containers.md

```diff
@@ -0,0 +1,18 @@
+title: Shipping Container Logs
+description: Shipping Container Log Sources
+
+To enable automatic container logs shipping, start by clicking on the `Set Up` button. This will open the following flyout panel that allows configuring container log shipping:
+
+![Enable Container Logs](images/enable-container-logs.png)
+
+1. Trigger the "Automatically ship <group-name> logs" toggle
+2. Select the destination Logs App. If the Logs App doesn't exist, you'll be able to create one from the same screen
+3. Click the `Save Changes` button
+
+Log shipping is configured by the specific container image. This implies that any freshly created containers that match this image name will automatically have their logs shipped to Sematext Cloud.
```

Comment:
Log shipping is configured by the specific container image.
=> 
Log shipping is configured on a per container image basis.